exporter/sumologicexporter/internal/observability/observability.go

```diff
@@ -0,0 +1,126 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package observability // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sumologicexporter/internal/observability"
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+func init() {
+	err := view.Register(
+		viewRequestsSent,
+		viewRequestsDuration,
+		viewRequestsBytes,
+		viewRequestsRecords,
+	)
+	if err != nil {
+		fmt.Printf("Failed to register sumologic exporter's views: %v\n", err)
+	}
+}
+
+var (
+	mRequestsSent     = stats.Int64("exporter/requests/sent", "Number of requests", "1")
```

Comment:
just one question is it possible to use otel instead of opencensus? it would be great not to add more opencensus dependencies. see https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/29867

Reply:
> just one question is it possible to use otel instead of opencensus?

I started looking at this, but seems it will require some refactor, so if it would be possible I would like to move the code from [our repo](https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/exporter/sumologicexporter) first 😅

Reply:
Can we create an issue to move from OpenCensus to OTel metrics?

Reply:
In this repo? I am not sure how to nice proceed with it as we are in middle of migration, and the opencensus code is not added to this repo yet 😅 
I can create it in advance, or we can work in our repo to get rid of it and then proceed with migration

Reply:
As I look at the example PRs I can see that opencensus goes from direct dependency to indirect dependency, but it is still used. e.g: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30736/files

Reply:
I created the issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/32261